examples/esp32spi_wsgiserver.py

```diff
@@ -0,0 +1,188 @@
+import os
+import board
+import busio
+from digitalio import DigitalInOut
+import neopixel
+
+from adafruit_esp32spi import adafruit_esp32spi
+import adafruit_esp32spi.adafruit_esp32spi_wifimanager as wifimanager
+import adafruit_esp32spi.adafruit_esp32spi_wsgiserver as server
+
+# This example depends on the 'static' folder in the examples folder
```

Comment:
Check for the existence of a static folder, or at least a file within `static/*.html`

See:
https://github.com/adafruit/Adafruit_CircuitPython_Display_Button/blob/master/examples/display_button_simpletest.py#L17

Reply:
Checking now before starting the server.